sdk/src/ed25519_instruction.rs

```diff
@@ -179,6 +187,7 @@ fn get_data_slice<'a>(
 
 #[cfg(test)]
 pub mod test {
+    use ed25519_dalek::Signer as DalekSigner;
```

Comment:
micro-nit: can you fold this `use` into the larger statement right under? Running `./cargo nightly fmt --all` from the repo root should take care of this for you

Reply:
`cargo fmt` also caught a one-liner in `sdk/program/src/serde_varint.rs` can I leave that in this PR even though the file is unrelated?

Reply:
yup, you're right, I was just running the latest normal `cargo +nightly` since I already had that installed and figured it'd do the trick just the same... Thanks for the explanation!

Reply:
with the wrapper script running clippy locally also finally works for me, I had been wondering what I was doing wrong there